package.json

```diff
@@ -9,36 +9,36 @@
     "commit": "git-cz",
     "checkcommit": "./scripts/commit-lint.js",
     "e2e": "./scripts/e2e.sh",
-    "format": "prettier \"./**/*.{ts,js,json,css,md}\" \"!./**/{__name__,__directory__}/**\" --write",
+    "format": "prettier \"**/*.{ts,js,json,css,md}\" \"!**/{__name__,__directory__}/**\" --write",
```

Comment:
I took off the relative path to fix the issue with `prettier` really wanting to check the tmp directory.

It possible could have to do with this issue https://github.com/prettier/prettier/issues/2964 which is closed but we are on an earlier version of `prettier`?